Lib/test/test_sys.py

```diff
@@ -1053,6 +1053,12 @@ def test_getallocatedblocks(self):
         c = sys.getallocatedblocks()
         self.assertIn(c, range(b - 50, b + 50))
 
+    def test_is_gil_enabled(self):
+        if support.Py_GIL_DISABLED:
+            self.assertIs(type(sys._is_gil_enabled()), bool)
```

Comment:
I'm don't understand why this is being tested.
Is there any chance that this can be non-boolean?

Reply:
It seemed useful to actually call the function even if there's not much we can say about the return value other than that it should be a `bool`.

Reply:
Basically, some bugs are uncovered just by a calling the function. Does it crash or is otherwise broken?

Reply:
Thanks for the clarifications, Sam!